packages/logs/README.md

```diff
@@ -117,16 +117,18 @@ window.DD_LOGS.init({
-| `env`                 | String  | No       |                 | The application’s environment, for example: prod, pre-prod, staging, etc.                                |
-| `version`             | String  | No       |                 | The application’s version, for example: 1.2.3, 6c44da20, 2020.02.13, etc.                                |
-| `forwardErrorsToLogs` | Boolean | No       | `true`          | Set to `false` to stop forwarding console.error logs, uncaught exceptions and network errors to Datadog. |
-| `sampleRate`          | Number  | No       | `100`           | The percentage of sessions to track: `100` for all, `0` for none. Only tracked sessions send logs.       |
-| `silentMultipleInit`  | Boolean | No       |                 | Prevent logging errors while having multiple init.                                                       |
+| Parameter                   | Type    | Required | Default         | Description                                                                                              |
+| --------------------------- | ------- | -------- | --------------- | -------------------------------------------------------------------------------------------------------- |
+| `clientToken`               | String  | Yes      |                 | A [Datadog client token][2].                                                                             |
+| `site`                      | String  | Yes      | `datadoghq.com` | The Datadog site of your organization. US: `datadoghq.com`, EU: `datadoghq.eu`                           |
+| `service`                   | String  | No       |                 | The service name for your application.                                                                   |
+| `env`                       | String  | No       |                 | The application’s environment, for example: prod, pre-prod, staging, etc.                                |
+| `version`                   | String  | No       |                 | The application’s version, for example: 1.2.3, 6c44da20, 2020.02.13, etc.                                |
+| `forwardErrorsToLogs`       | Boolean | No       | `true`          | Set to `false` to stop forwarding console.error logs, uncaught exceptions and network errors to Datadog. |
+| `sampleRate`                | Number  | No       | `100`           | The percentage of sessions to track: `100` for all, `0` for none. Only tracked sessions send logs.       |
+| `silentMultipleInit`        | Boolean | No       |                 | Prevent logging errors while having multiple init.                                                       |
+| `useAlternateIntakeDomains` | Boolean | No       | `false`         | Use the new intake domains for sending Logs data. Make sure to update [CSP][6] headers before enabling.  |
+| `intakeApiVersion`          | Number  | No       | `1`             | Set to `2` to use the new intake API for sending Logs data.                            |
```

Comment:
Should we document it? It could be a bit confusing for customers and afaik enabling that offer no value for them